regolith/helpers/a_proposalhelper.py

```diff
@@ -0,0 +1,190 @@
+"""Helper for adding a proposal to the proposals.yml collection.
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+)
+
+TARGET_COLL = "proposals"
+
+def subparser(subpi):
+    # Do not delete --database arg
+    subpi.add_argument("--database",
+                       help="The database that will be updated.  Defaults to "
+                            "first database in the regolithrc.json file."
+                       )
+    subpi.add_argument("name", help="A short but unique name for the proposal",
+                       )
+    subpi.add_argument("amount", help="value of award",
+                       )
+    subpi.add_argument("title", help="Actual title of the proposal"
+                       )
+    subpi.add_argument("--begin_date", help="The begin date for the proposed grant "
+                                          "in format YYYY-MM-DD.",
+                       default = 'tbd'
+                       )
+    subpi.add_argument("-d", "--duration", help="Duration of proposal in months",
+                       default = 'tbd'
+                       )
+    subpi.add_argument("--due_date", help="The due date for the proposal in format YYYY-MM-DD.",
+                       default = 'tbd'
+                       )
+    subpi.add_argument("-a", "--authors", nargs="+",
+                       help="Other investigator names", default = []
+                       )
+    subpi.add_argument("-c", "--currency",
+                       help="Currency in which amount is specified. Defaults to USD",
+                       default = 'USD'
+                       )
+    subpi.add_argument("-p", "--pi",
+                       help="ID of principal investigator. Defaults to"
```

Comment:
you will need a whitespace at the end of this line for this to display correctly

Reply:
done!